### PR TITLE
Update v1.1 Docker Swarm instructions to use the init command

### PR DESCRIPTION
**Pull request by a-robinson**
Also switch to using an `attachable` network which makes running one-off
containers easier and simplify/fix the docs as applicable.

We could also go a little further with the secure guide and remove the
client keys from each container now that we can connect to the cluster
with one-off containers, but it'll probably still come in handy in
debugging situations to have the keys in all the containers.

Fixes part of https://github.com/cockroachdb/docs/issues/1288. I'm not yet making the change suggested in https://github.com/cockroachdb/cockroach/issues/17133 because it would make our insecure and secure setups diverge quite a bit, and I'm not thrilled about that.

### CODE REVIEW

**Comment by a-robinson** [comment]
I tested this using the alexrobinson/cockroach:v1.1-beta.20170907 docker image, in case anyone wants to try it themselves.

File: v1.0/orchestrate-cockroachdb-with-docker-swarm.md [hunk 1]
```diff
@@ -528,7 +528,7 @@ To increase the number of nodes in your CockroachDB cluster:
 4. Create security resources for the node (see [Step 5.7 and 5.8](#step-5-create-security-resources)).
 5. Create a new service to start another node and join it to the CockroachDB cluster (see [Step 6.2](#step-6-start-the-cockroachdb-cluster)).
 
-## Step 10. Stop the cluster
+## Step 11. Stop the cluster
```
Comment (jseldess) [inline v1.0/orchestrate-cockroachdb-with-docker-swarm.md]: Thanks for catching these, @a-robinson!

File: v1.1/orchestrate-cockroachdb-with-docker-swarm-insecure.md [hunk 2]
```diff
@@ -197,57 +201,31 @@ $ sudo docker network create --driver overlay cockroachdb
...
-    ~~~ shell
-    $ sudo docker ps | grep cockroachdb
-    ~~~
-
-    ~~~
-    9539871cc769        cockroachdb/cockroach:{{page.release_info.version}}   "/cockroach/cockroach"   2 minutes ago        Up 2 minutes         8080/tcp, 26257/tcp   cockroachdb-1.1.0wigdh8lx0ylhuzm4on9bbldq
-    ~~~
-
-2. Use the `sudo docker exec` command to open the built-in SQL shell in interactive mode inside the container:
+1. Use the `sudo docker run` command to run a new container attached to the CockroachDB network and connect to the cluster:
```
Comment (jseldess) [inline v1.1/orchestrate-cockroachdb-with-docker-swarm-insecure.md]: I think it's helpful to call out that we're launching the built-in SQL shell, but I see that you removed that language. Do you feel that "connect to the cluster" conveys the same thing?

Comment (a-robinson) [reply v1.1/orchestrate-cockroachdb-with-docker-swarm-insecure.md]: You're right, being more specific about how we're connecting is better. I've changed these back.

**Pull request merged**